drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -204,7 +183,7 @@ def do_attempt(self, attempt, suffix=''):
                 # If this item has no input value set, we are done with it.
                 is_correct = True
                 feedback = item['feedback']['correct']
-            state = {'top': attempt['top'], 'left': attempt['left']}
+            state = {'top': attempt['top'], 'left': attempt['left'], 'position': 'absolute'}
```

Comment:
@antoviaque Makes sense, I"ll replace `'position': 'absolute'` with `'absolute': True` and add a comment explaining why the flag is needed.